src/main/java/com/checkmarx/sdk/service/FilterValidator.java

```diff
@@ -128,10 +129,15 @@ private Double getNumericScore(List<String> filterValues) {
         if (!CollectionUtils.isEmpty(filterValues)) {
             if (filterValues.size() == 1) {
                 String scoreString = filterValues.get(0);
-                try {
-                    result = numberFormat.parse(scoreString).doubleValue();
-                } catch (ParseException e) {
-                    log.warn("Invalid {} filter value: '{}', ignoring.", Filter.Type.SCORE, scoreString);
+                if(scoreString.equals(EMPTY_STRING)){
```

Comment:
Shouldn't it be the other way round? if(!scoreString.equals(EMPTY_STRING))
Also it's better to use StringUtils.isNotEmpty.